flaskbb/templates/forum/topic.html

```diff
@@ -147,14 +147,14 @@
                             {% endif %}
 
                             {% if current_user.permissions.get('makehidden') %}
-                                {% if post.first_post %}
+                                {% if topic.first_post_id == post.id %}
```

Comment:
Not sure what my original intention with `post.first_post` was here since that doesn't actually appear to be a thing???

Reply:
Haha yeah, I had to look it up and it appears to work because of the backref on Topic.first_post